be/src/runtime/fragment_mgr.cpp

```diff
@@ -786,7 +786,7 @@ Status FragmentMgr::exec_plan_fragment(const TExecPlanFragmentParams& params,
     std::shared_ptr<RuntimeFilterMergeControllerEntity> handler;
     // TODO need check the status, but when I add return_if_error the P0 will not pass
     static_cast<void>(_runtimefilter_controller.add_entity(
```

Comment:
return if error?

Reply:
zhiqiang hhh has tested, it will core .... but I donot know why